src/core/rendering.js

```diff
@@ -119,6 +119,20 @@ p5.prototype.createCanvas = function(w, h, renderer) {
   }
   this._renderer.resize(w, h);
   this._renderer._applyDefaults();
+
+  //isWebGL flag is set and made read-only
+  if (this.isWebGL == false) {
+    Object.defineProperty(this,"isWebGL" , {
+      value: this._renderer.drawingContext instanceof WebGLRenderingContext,
+      writable: false
+    });
+    }
```

Comment:
Also, it looks like the spacing and equal signs aren't quite lined up here. Can you double check that `npm run lint` passes after making changes?